submissions/AndriyBerezovskiy/hooli-style-popup/index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>popup</title>
+    <link rel="icon" type="image/x-icon" href="./img/favicon.svg" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__menu">
+        <ul class="header__list df">
+          <li>
+            <a class="header__link" href="#">Mail</a>
+          </li>
+          <li>
+            <a class="header__link" href="#">Images</a>
+          </li>
+          <li class="popup">
+            <input type="checkbox" id="popup__input" class="popup__input" />
```

Comment:
Add `cursor: pointer` to improve user experience

Reply:
Add `cursor: pointer` where it wasn't. My mistake)